spikesorters/basesorter.py

```diff
@@ -138,51 +138,22 @@ def run(self, raise_error=True, joblib_backend=None):
                     self.parallel = False
                     print("RecordingExtractor objcts are not dumpable and can't be processed in parallel")
 
-        if raise_error:
+        try:
```

Comment:
I did this raise_error in purpose because it was raising the real error.
Wrapping our own error with SpikeSortingError is not a so good idea because it was harder the debugging when there is a problem with a sorter. The original error is easier.
This must be discuss I think

Reply:
Ok